src/components/InputDynamicArray.tsx

```diff
@@ -62,8 +64,12 @@ export default class FieldDynamicArray extends React.Component<FieldDynamicArray
   }
 
   render() {
+    const t = this.props.t;
     const inputs = this.values.map((v, i) => {
-      const deleteValueBtn= <DeleteValueInputButton onClick={this.deleteValue.bind(this, i)} />
+      const deleteValueBtn= <DeleteValueInputButton 
+        onClick={this.deleteValue.bind(this, i)}
+        {...{t, i18n: this.props.i18n, tReady: this.props.tReady}}
```

Comment:
What's the use of the `{...{`? Is there a simpler way to write this?

Reply:
`{...` is basically React's syntax for "the rest of the props for this component should come from this object.

The alternative would be to A) specify each property manually (`t={t} i18n={this.props.i18n}...`), B) create a `i18nProps` variable like `const i18nProps = {t, i18n: .., tReady: ..}; ...<component {...i18nProps}>`, or C) wrap each component in a HOC. If we were using function components, I wouldn't have a problem using the `useTranslation` hook in each component, but HOCs are a bit unwieldily, so I decided to just passthrough the translation props from the parent.

I'll clean it up by making `i18nProps` (B) for now; let me know if there's a better way to clean it up